### ISSUE TITLE
Adding profile views 

### ISSUE THREAD

**Issue by root-0101** [bishalde/GithubCrawler#32]
It would be great if we also display the no of profile views 

![Screenshot (85)](https://github.com/bishalde/GithubCrawler/assets/142779809/6b6a4135-4293-409e-a5a8-92d579fac4e0)

**Comment by gyanendra-baghel** [comment]
I think github api does not provide, number of profile views.

**Comment by root-0101** [comment]
Yes, but some alternative is there if you allow me can I go for this issue
